README.md

```diff
@@ -70,6 +82,10 @@ Example Playbooks
       log_level: INFO
       apm_enabled: "true" # has to be set as a string
       logs_enabled: true  # log collection is available on agent 6
+      process_config: # on agent 6
```

Comment:
I think all of this part is for agent 6?

Reply:
The conf file also uses `datadog_config` to populate the template and the yaml template doesn't use the `datadog_config_ex` that's is presented on the Playbook Example. That's why I've put this comment here.